intel_extension_for_transformers/transformers/modeling/modeling_auto.py

```diff
@@ -154,6 +154,7 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
                     use_ggml=quantization_config.use_ggml,
                     not_quant=quantization_config.not_quant,
                     use_cache=quantization_config.use_cache,
+                    from_gptq=quantization_config.from_gptq,

```

Comment:
@xin3he do we usually have ```from_gptq``` field in the config?

Reply:
Shall we unify the parameter? e.g., not_quant -> use_quant, and update the logic accordingly; from_gptq -> use_gptq?